### DISCUSSION TITLE
Share your settings here!

### DISCUSSION THREAD

**Discussion by bhackel** [arichornlover/uYouEnhanced#651] [Show and tell]
This post is a place to share your custom settings, which can be imported/exported using the `Copy Settings` and `Paste Settings` features.

Here is an example to get this thread started:
### Unhook
This setting pack is designed to remove any suggested videos anywhere, so that you only have your subscriptions. It will hide the home tab, hide suggested videos under the player, hide the shorts tab, and includes a few other UI tweaks.
```
hideHomeTab_enabled: 1
hideShortsTab: 1
hideCreateTab: 1
noSuggestedVideo_enabled: 1
noRelatedWatchNexts_enabled: 1
startupPage: FEsubscriptions
hideFullscreenActions_enabled: 1
inline_muted_playback_enabled: 1
```

**Comment by 54437698** [comment]
I just wanted to say thanks to everyone in development who participates in making this project and bringing it back to life, I used to have fun doing these things myself but I'm very disabled now and I'm unable to do it and I just wanted to let everybody know that I want to express my deepest gratitude for what everyone in the community and this project is doing here, thanks everyone.

**Comment by arichornlover** [comment]
Hey everyone, what are your thoughts on the new option called `replaceCopyandPasteButtons_enabled` when enabled, it changes the buttons to use a .txt file instead of using the clipboard.
https://github.com/arichornlover/uYouEnhanced/commit/0ea26e29b8703d640b9a313413a0cc0add3a03c7

**Reply by bhackel** [reply]
> Hey everyone, what are your thoughts on the new option called `replaceCopyandPasteButtons_enabled` when enabled, it changes the buttons to use a .txt file instead of using the clipboard. [0ea26e2](https://github.com/arichornlover/uYouEnhanced/commit/0ea26e29b8703d640b9a313413a0cc0add3a03c7)

I think it could be useful if someone wanted to update the app - they could export their settings to a file, reinstall the app, and then quickly load the same file back into the app.
